service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/LegalRepresentativeAddedContentProviderTest.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.CaseUrlService;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@ExtendWith(MockitoExtension.class)
+class LegalRepresentativeAddedContentProviderTest {
+
+    public static final String REPRESENTATIVE_FULL_NAME = "representative full name";
+    public static final LegalRepresentative LEGAL_REPRESENTATIVE = LegalRepresentative.builder()
+        .fullName(REPRESENTATIVE_FULL_NAME)
+        .build();
+    public static final String LOCAL_AUTHORITY = "Local Authority";
+    public static final String RESPONDENT_LAST_NAME = "RepspondentLastName";
+    public static final Element<Respondent> ANOTHER_RESPONDENT = element(mock(Respondent.class));
```

Comment:
is there a reason for this mock?

Reply:
this is to assert that is the first element of the list if there are multiple RESPONDANT : see firstRespondentLastName